package.json

```diff
@@ -26,7 +26,7 @@
     "type": "git",
     "url": "https://github.com/sciencemesh/cs3api4lab.git"
   },
-  "scripts": { 
```

Comment:
Please do not change this file. This change is converting 4 spaces in a mix of 4 and 2.